.env.example

```diff
@@ -1,3 +1,5 @@
-CLIENT_ID=grueneapp
-ISSUER=https://keycloak.example.com/realms/grueneapp
+APP_ID=com.example.grueneapp
+OIDC_CALLBACK_PATH=oauthredirect
```

Comment:
Sorry, I think I did not communicate well what my thought was. I did not intend to make these vars configurable or part of the env.
Instead, I just wanted to have those as flutter constants somewhere in the code without being set in the env.
Sorry for the miscommunication here.